master/buildbot/reporters/http.py

```diff
@@ -29,7 +29,15 @@ class HttpStatusPush(ReporterBase):
     secrets = ["auth"]
 
     def checkConfig(
-        self, serverUrl, auth=None, headers=None, debug=None, verify=None, generators=None, **kwargs
+        self,
+        serverUrl,
+        auth=None,
+        headers=None,
+        debug=None,
+        verify=None,
+        skipEncoding=False,
```

Comment:
Let's use snake_case for new code.

Reply:
oh, I did not realize the snake case was intentional. thanks